encoding/protobuf/error_integration_test.go

```diff
@@ -40,14 +38,16 @@ import (
 	"go.uber.org/yarpc/encoding/raw"
 	"go.uber.org/yarpc/transport/grpc"
 	"go.uber.org/yarpc/yarpcerrors"
+	rpcStatus "google.golang.org/genproto/googleapis/rpc/status"
```

Comment:
nit: let's rename this statuspb, even if spb is kind of the same it will help for readability of the code. Also make sure that it is the same name in yarpc everywhere